src/app/pages/system/ca/ca-add/ca-add.component.ts

```diff
@@ -27,7 +27,9 @@ export class CertificateAuthorityAddComponent {
       placeholder : T('Identifier'),
       tooltip: T('Enter a description of the CA.'),
       required: true,
-      validation : [ Validators.required ]
+      validation : [ Validators.required, Validators.pattern('[A-Za-z0-9_-]+$') ],
+      hasErrors: false,
+      errors: 'Use alphanumeric characters, "_" and "-".',
```

Comment:
The quotes can imply that quote marks can be used in the field.  Suggested rewording:
` errors: 'Allowed characters: letters, numbers, underscore (_), and dash (-).', `